### ISSUE TITLE
Ingame Overlay ingame is not working anymore

### ISSUE THREAD

**Issue by feiercrack** [Marginal/EDMarketConnector#422]
since last december i cant see the ingame overlay anymore...

[error.txt](https://github.com/Marginal/EDMarketConnector/files/3214346/error.txt)

**Comment by inorton** [comment]
Hi @feiercrack I'm the author of EDMC Overlay (separate project - https://github.com/inorton/EDMCOverlay)   I'm going to guess that you are using EDR? If so you might need to make sure the "overlay" checkbox is enabled in the EDR settings page

**Comment by feiercrack** [comment]
Hello @inorton 
thank you for the response.
Sure i have checked both boxes in EDMC. Also i updated all the plugins and the EDMC.
I also checked that EDR now contains your plugin.
Then i tried to rollback to older versions.
but i simply cant get it to run on my main screen.
EDMC is running fine on my 2nd monitor next to me. also i can see my orders in the text-box below inside EDMC. 
but i saw also, that i cant login anymore at EDR, they had changed something about the access...
dunno why that is...(i have 1000+hrs in ED)
i added the log file above. in there u can see that it suddenly stops to work on 2018-12-11T22:18:24 without any reason.
i was playing the whole night. so there was no graphics card update or anything else kinda that...
i am running Win 10 x64 -1809, asus 1070 8gb with latest nvidia driver 430.64. i7-7700k on asus apex ix z270, G.Skill Trident Z RGB DIMM Kit 16GB, DDR4-4133 on Samsung SSD 850 EVO 500GB, SATA, HTC VIVE, TrackIR and Thrustmaster Warthog HOTAS, Logitech Keyboard and Mouse. ESET Smart Security Premium 12 Firewall and iNet Security.
Could there be an problem with my other plugins?

![](https://i.gyazo.com/abe73097b096b3a577e308f98401507c.png)

Update:

some things can be very easy, you only havent think about, cause its too easy ;-) 
 I tried to run EDMC with admin priviledges, now it works again like a charm!
Thanks anyways.. but maybe this helps other user!

keep on going the good stuff :-D 
cheers from germany,
CMDR Feiercrack o7
